inbc-program/inbc/command/config_command.py

```diff
@@ -9,7 +9,7 @@
 from pathlib import Path
 from .command import Command
 from ..utility import copy_file_to_target_location, search_keyword
-from ..constants import COMMAND_SUCCESS, COMMAND_FAIL, MAX_TIME_LIMIT
+from ..constants import COMMAND_SUCCESS, COMMAND_FAIL, MAX_TIME_LIMIT, TC_INSTALL_CHANNEL
```

Comment:
Let's not use the initials TC.  Use INBM instead.  We're trying to remove all references of TC and BC since they are code names.